conf.d/gitnow.fish

```diff
@@ -245,6 +245,22 @@ function release -d "GitNow: Creates a new Gitflow release branch from current b
     commandline -f repaint
 end
 
+function merge -d "GitNow: Merges given branch into the active one"
+    if not __gitnow_is_git_repository
+        __gitnow_msg_not_valid_repository "merge"
+        return
+    end
+
+    set -l len (count $argv)
+    set -l opts .
```

Comment:
Since the default argument looks like is a dot, a simple `merge` (`git merge .`) execution throws an error like `fatal: . - not something we can merge`.
If some argument is needed, I suggest to add some checks.